runtime/src/main/resources/application-zitadel.properties

```diff
@@ -0,0 +1,22 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+#
+# spring security oauth2 config
+logging.level.org.springframework.web=info
+logging.level.org.springframework.security=debug
+# set authorization server endpoint
+authorization-server-endpoint=https://iam.xpanse.site
+authorization-api-client-id=221665786966638595@eclipse-xpanse
+authorization-api-client-secret=2S7yZR4jWauyE3wLHM1h5asc0WNWdBUXAw2Ill3Pmu0qM4D3LWBCwikXbsG81ycl
+spring.security.oauth2.resourceserver.opaquetoken.introspection-uri=${authorization-server-endpoint}/oauth/v2/introspect
+spring.security.oauth2.resourceserver.opaquetoken.client-id=${authorization-api-client-id}
+spring.security.oauth2.resourceserver.opaquetoken.client-secret=${authorization-api-client-secret}
+# springdoc openapi security oauth2 config
+authorization-useragent-client-id=221664356859969539@eclipse-xpanse
+springdoc.show-login-endpoint=true
+springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
+springdoc.swagger-ui.oauth.clientId=${authorization-useragent-client-id}
```

Comment:
we can directly assign the value here.

Reply:
@baixinsui  ok. then we can keep it as it is. Just that we can replace the `authorization-useragent-client-id` with `authorization-swagger-ui-client-id`

Reply:
OK.